readme.md

```diff
@@ -340,6 +340,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Apdex](https://apdex.co)** - Apdex is a website performance service. The main features are historical Lighthouse report visualizations, mobile/desktop options, alerts, uptime monitoring, and more. There are flexible paid plans and a 30-day free trial.
 
+* **[Websu](https://websu.io)** - Websu is an [open source project](https://github.com/websu-io/websu) to provide Lighthouse-as-a-Service through a simple HTTP REST API. The main features are ability to host and deploy in your own environment and historical Lighthouse report summaries.
```

Comment:
@samos123 fwiw, we only allow 1 link for these. which you rather link to?

if its the github, we'd put this in the `## Related Projects` section.  

if it's  the .com, then the placement here is cool. you can drop the gh link.

Reply:
Thank you I will remove the github link since that's also linked to from the main website.